src/sentry/replays/blueprints/api.md

```diff
@@ -0,0 +1,366 @@
+# Replays API
+
+Host: https://sentry.io/api/0
+
+**Authors.**
+
+@cmanallen
+@joshferge
+
+**How to read this document.**
+
+This document is structured by resource with each resource having actions that can be performed against it. Every action that either accepts a request or returns a response WILL document the full interchange format. Clients may opt to restrict response data or provide a subset of the request data. The API may or may not accept partial payloads.
+
+## Replays [/organizations/<organization_slug>/replays/]
+
+- Parameters
+
+  - field (optional, string)
+  - environment (optional, string)
+  - project (optional, string)
+  - sort (optional, string)
+    Default: -startedAt
+    Members: + projectId + -projectId + startedAt + -startedAt + finishedAt + -finishedAt + duration + -duration + countErrors + -countErrors
+  - statsPeriod (optional, string) - A positive integer suffixed with a unit type.
+    Default: 7d
+    Members: + s + m + h + d + w
+  - start (optional, string) - ISO 8601 format (`YYYY-MM-DDTHH:mm:ss.sssZ`)
+  - end (optional, string) - ISO 8601 format. Required if `start` is set.
```

Comment:
Can we default to `now` if this is `start` omitted?
```suggestion
  - end (optional, string) - ISO 8601 format. Defaults to `now`.
```

Reply:
I'm porting the blueprints from replay-backend.  If you want to request changes (unrelated to DOM search) we should open an issue to discuss.

Reply:
Edited the description to help clarify that:

> This PR ports blueprints from https://github.com/getsentry/replay-backend/ while adding new things, related to the DOM search feature: https://github.com/getsentry/sentry/issues/45524